modelskill/skill.py

```diff
@@ -521,9 +519,17 @@ def __getitem__(self, key) -> SkillArray | SkillTable:
     def __getattr__(self, item):
         if item in self.data.columns:
             return self[item]  # Redirects to __getitem__
+        else:
+            raise AttributeError(f"SkillTable has no attribute {item}")
```

Comment:
alternatively, we could do 

    attr = getattr(self.data, item)
    if (isinstance(attr, pd.DataFrame))
      return SkillTable(attr) # create a new SkillTable object (in try-except?)